apps/studio/src/features/editing-experience/components/form-builder/components/LinkErrorBoundary.tsx

```diff
@@ -0,0 +1,50 @@
+import { PropsWithChildren } from "react"
+import { IconButton, Stack, Text } from "@chakra-ui/react"
+import { Infobox } from "@opengovsg/design-system-react"
+import { ErrorBoundary } from "react-error-boundary"
+import { BiTrash } from "react-icons/bi"
+
+interface LinkErrorBoundaryProps {
+  resetLink: () => void
+}
+export const LinkErrorBoundary = ({
+  resetLink,
+  children,
+}: PropsWithChildren<LinkErrorBoundaryProps>) => {
+  return (
+    <ErrorBoundary
+      fallbackRender={({ resetErrorBoundary }) => (
+        <Infobox
+          variant="error"
+          borderRadius="4px"
```

Comment:
nit - use 1rem instead since we have mostly been using rem over px

Reply:
oh yes! good catch! have to talk to @sehyunidaaa regarding this because there's a [difference](https://austingil.com/px-or-rem-in-css/) - but i think since we are already using `rem` for borders, might as well continue omegalull